images/macos/provision/core/xcode.ps1

```diff
@@ -15,6 +15,9 @@ if ([string]::IsNullOrEmpty($env:XCODE_INSTALL_USER) -or [string]::IsNullOrEmpty
 $env:SPACESHIP_SKIP_2FA_UPGRADE = 1
 
 $os = Get-OSVersion
+# # Monterey needs future review:
+# For now we install only Xcode 13.0
+# https://github.visualstudio.com/virtual-environments/_build/results?buildId=115815&view=results
```

Comment:
Please remove this comment completely. It is expected that Monterey doesn't support old Xcode versions and no other actions are required here.

Reply:
Done. Thank you.